reporter/src/funTest/assets/notice-summary-reporter-expected-results

```diff
@@ -0,0 +1,55 @@
+This project contains or depends on third-party software components pursuant to the following licenses:
```

Comment:
Commit message: Maybe `Use the same test data as the "by package" reporter`?

Reply:
Done.

Reply:
Almost 😉 You've omitted the "the"s 😆 Approved anyway.